templates/includes/documentation-header.hbs

```diff
@@ -4,11 +4,7 @@
     <h1>
     {{#is slug 'documentation'}}
     {{else}}
-      <a href="{{rel 'wiki/documentation'}}">Documentation</a> <svg width="4.3482mm" height="6.0193mm" version="1.1" viewBox="0 0 6.9570877 9.6308298">
-<g transform="translate(-73.598 -103.17)">
-<g transform="translate(1.6)" fill="#666">
-<path d="m73.598 103.17h0.85725l2.8998 4.8154-2.8998 4.8154h-0.85725l2.8892-4.8154z" fill="#666" stroke-width=".26458"/>
-</g></g></svg> {{/is}}
+      <a href="{{rel 'wiki/documentation'}}">Documentation</a> ⟩ {{/is}}
```

Comment:
I like it, but that unicode symbol might be hard to spot for people doing maintenance on these handlebars files.  Might it be better to use `&#x27e9;` instead?

Reply:
On the other hand, sources I've found recommend using the character directly if the encoding is UTF-8.  It doesn't look like we explicitly set the encoding, however, so I'm not sure.

Reply:
Agreed, bad discoverability while maintaining is a striking argument.